reportportal_client/helpers.py

```diff
@@ -11,14 +11,47 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-
 import logging
-from pkg_resources import DistributionNotFound, get_distribution
+import uuid
 from platform import machine, processor, system
 
+from pkg_resources import DistributionNotFound, get_distribution
```

Comment:
This one is built-in and should be mentioned in the first import block.

Reply:
Mm, actually this line was replaced by pycharm auto import correct feature. If you do not to manually oput it on some specific place after each ctrl+alt+o would suggest to leave as is.